src/resources/app-en.json

```diff
@@ -1215,14 +1215,25 @@
   "tag": "Tag",
   "takeMeToTheResponseData": "Take me to the response data",
   "takePhoto": "Take Photo",
-  "takeNow": "Take Now",
```

Comment:
This is causing other buttons to miss text 
![CleanShot 2024-05-21 at 18 51 29](https://github.com/ChildMindInstitute/mindlogger-admin/assets/2916156/46de0173-74c1-49d1-bd08-64139eae0c8c)

Reply:
Assuming this is the button in `RespondentDataHeader`. Fixed it. Thanks